javascript/src/test/components/Admin/LoginTable.test.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import LoginTable from "main/components/Admin/LoginTable";
+
+describe("LoginTable tests", () => {
+
+    test("renders without crashing on empty list", () => {
+        render(<LoginTable loginTable={[]} />);
+    });
+
+
+  test("renders without crashing on non empty list", () => {
+
+    const loginTable = [
+        {
+            "timestamp": "7:00",
```

Comment:
Timestamp data looks slightly different in mongodb collections but I don't think it'll impact anything.
```suggestion
            "timestamp": "2020-12-08 14:41:48.481 -0800",
```